apps/cookies-test/src/app/app.component.ts

```diff
@@ -63,6 +67,13 @@ export class AppComponent {
 	ngOnInit() {
 		this.ngxCookieService.setCookie({ name: 'authenticated', value: true });
 		this.ngxCookieService.setCookie({ name: 'kbo', value: true });
+
+		this.sessionService.storageEvents$.subscribe(console.log);
```

Comment:
Is the `cookies-test` application the right one for these examples? 🤔

Reply:
TBF, these apps aren't meant for examples, that's why they have a disclaimer at the top 😂 They're just used to test stuff for ourselves.